CHANGELOG.rst

```diff
@@ -9,6 +9,8 @@ Unreleased - YYYY-MM-DD
 
 Features and Enhancements
 -------------------------
+- Add leader status to the ``synapse.tools.aha.list`` tool output. 
```

Comment:
Do we want to note the leader status is only displayed for cells where a leader has been registered?

Reply:
Yeah would probably be good to include that, updated